app/assets/stylesheets/govuk_publishing_components/components/_highlight-boxes.scss

```diff
@@ -2,56 +2,56 @@
   @include govuk-text-colour;
   display: flex;
   flex-wrap: wrap;
-  margin: $gutter-half 0 0 0;
+  margin: govuk-spacing(3) 0 0 0;
   padding: 0;
 
-  @include media(tablet) {
+  @include govuk-media-query($from: tablet) {
     margin-right: -25px;
   }
 }
 
 .gem-c-highlight-boxes__item-wrapper {
-  @include core-19;
-  @include box-sizing(border-box);
+  @include govuk-font(19);
+  box-sizing: border-box;
```

Comment:
Is this what we should be using now? There's no mixin for it?

Reply:
No mixin in govuk-frontend for `box-sizing`, the attribute is supported since IE8